src/Native/Bootstrap/platform.unix.cpp

```diff
@@ -104,4 +104,9 @@ extern "C"
     {
```

Comment:
You may be able to delete GetNativeSystemInfo from this file - it should not be needed anymore.